parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -15,6 +15,19 @@ TELEPORTD_PLUGIN_DOWNLOAD_DIR="/opt/teleportd/downloads"
 TELEPORTD_PLUGIN_BIN_DIR="/usr/local/bin"
 KRUSTLET_VERSION="v0.0.1"
 MANIFEST_FILEPATH="/opt/azure/manifest.json"
+MAN_DB_AUTO_UPDATE_FLAG_FILEPATH="/var/lib/man-db/auto-update"
+
+removeManDbAutoUpdateFlagFile() {
+    if [ -f "$MAN_DB_AUTO_UPDATE_FLAG_FILEPATH" ]; then
+        rm $MAN_DB_AUTO_UPDATE_FLAG_FILEPATH
+    fi
+}
+
+createManDbAutoUpdateFlagFile() {
+    if [ ! -f "$MAN_DB_AUTO_UPDATE_FLAG_FILEPATH" ]; then
+        touch $MAN_DB_AUTO_UPDATE_FLAG_FILEPATH
```

Comment:
```suggestion
        touch $MAN_DB_AUTO_UPDATE_FLAG_FILEPATH
```

touch won't error if the file already exists, so it's up to you, but we don't really need to have that inside an if check at all.